src/v/cluster/rm_stm.cc

```diff
@@ -334,6 +334,22 @@ rm_stm::rm_stm(
     auto_abort_timer.set_callback([this] { abort_old_txes(); });
     _log_stats_timer.set_callback([this] { log_tx_stats(); });
     _log_stats_timer.arm(clock_type::now() + _log_stats_interval_s());
+    set_bootstrap_committed_fiber();
+}
+
+void rm_stm::set_bootstrap_committed_fiber() {
+    ssx::spawn_with_gate(_gate, [this]() {
+        return bootstrap_committed_offset()
+          .then([this](model::offset o) { _bootstrap_committed_offset = o; })
+          .handle_exception([this](const std::exception_ptr& e) {
```

Comment:
nit: I think we can skip warn log if e is of type `ss::abort_requested_exception`

Reply:
probably gate closed as well, i will add